README.md

```diff
@@ -79,6 +79,7 @@ The highest stat is "strength"! Amazing!
 |  cslib_menu  | Routines to generate menus and choices. |  1.0.0  |           |                                                            |
 |  cslib_loop  |   Repeat the execution of a routine.    |  1.0.0  |            |                           repeat                           |
 |  cslib_array  |  Various array manipulation.  |  1.0.0  |    |  filter, clone, clean, push, pop, etc.  |
+|  cslib_math   | Trigonometric and hyperbolic functions | 1.0.0 | sin, cos, sinh, cosh, pi |
```

Comment:
> Trigonometric and hyperbolic functions

Can you think of something more generic? This might grow in time.
E.g. I'd consider adding floor and ceil.